packages/shared/src/server/services/S3StorageService.ts

```diff
@@ -40,10 +39,11 @@ export class S3StorageService {
 
     this.client = new S3Client({
       credentials,
-      endpoint,
-      region,
+      endpoint: params.endpoint,
+      region: params.region,
+      forcePathStyle: params.forcePathStyle,
```

Comment:
`forcePathStyle` is a required variable to make the S3 client minio compatible. Useful for local deploys and self-hosters.